### ISSUE TITLE
StatusBarOverlaysWebView ignoring from  preference in latest version of android too

### ISSUE THREAD

**Issue by to-svarghese** [apache/cordova-plugin-statusbar#209]
It is said  in the documentation that,  <preference name="StatusBarOverlaysWebView" value="true" /> will be default for Android versions higher than 5. But in real it is  not taking the default value  or not even if we set  <preference name="StatusBarOverlaysWebView" value="true" /> in config.

I had to use  StatusBar.overlaysWebView(true); onDeviceReady to make the overlay true in Android.

Cordova-plugin-statusbar: 2.4.3

**Comment by breautek** [comment]
Can you take a moment to test the development version of the plugin? I believe this would be resolved by https://github.com/apache/cordova-plugin-statusbar/pull/171 which hasn't been released yet.

You can install the development version directly from git by using:

`cordova plugin add https://github.com/apache/cordova-plugin-statusbar`

**Comment by mosabab** [comment]
I agree with @to-svarghese that the preference (StatusBarOverlaysWebView) not work.

Only the current temp solution is to use:
`StatusBar.overlaysWebView(true);`

**Comment by CodeWithOz** [comment]
@breautek are there any changes that can be made in light of [this development](https://piunikaweb.com/2021/01/07/did-samsung-one-ui-3-0-android-11-update-remove-option-to-hide-camera-cutout/)? That PR works only if the camera cutout is hidden on the Samsung devices mentioned in that article. Otherwise the statusbar is still shifted down further than desired.
